messaging/kafka-avro-reactive-messaging/src/test/java/io/quarkus/ts/openshift/messaging/kafka/AbstractKafkaTest.java

```diff
@@ -0,0 +1,71 @@
+package io.quarkus.ts.openshift.messaging.kafka;
+
+import io.quarkus.ts.openshift.common.injection.TestResource;
+import org.junit.jupiter.api.MethodOrderer;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestMethodOrder;
+
+import javax.ws.rs.client.Client;
+import javax.ws.rs.client.ClientBuilder;
+import javax.ws.rs.client.WebTarget;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.sse.SseEventSource;
+
+import java.net.URL;
+import java.util.List;
+import java.util.concurrent.CopyOnWriteArrayList;
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.TimeUnit;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+public abstract class AbstractKafkaTest {
+
+    private static final int TIMEOUT_SEC = 25;
+    private static final int EVENTS_AMOUNT = 3;
+
+    private String endpoint;
+    private Client client = ClientBuilder.newClient();
+    private List<String> receive = new CopyOnWriteArrayList<>();
+    private boolean completed;
+
+    @TestResource
+    private URL sseEndpoint;
+
+    @Test
+    @Order(1)
```

Comment:
Do we need special order handling for a single test? Or maybe is this needed for something else?